herddb-core/src/main/java/herddb/model/Table.java

```diff
@@ -152,6 +155,22 @@ public boolean isPrimaryKeyColumn(int index) {
         return primaryKeyProjection[index] >= 0;
     }
 
+    public ForeignKeyDef[] getForeignKeys() {
+        return foreignKeys;
+    }
+
+    public boolean isChildTable(String parentUuid) {
```

Comment:
Can we change to isChildOf()? I was unsure between: "this table is child of given table and given table is child of this"

Reply:
renamed to isChildOfTable